CHANGELOG.md

```diff
@@ -8,7 +8,19 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
----
+## [0.13.0] - 2024-11-04
+
+### Added
+
+- Add support for Kubernetes namespace labels in the Drupal projects configuration
+
+### Changed
+
+- **BREAKING CHANGE**: We have removed the `lifecycle.ignore_changes` block from the `kubernetes_namespace` resource configuration. External modifications on namespace labels will now cause drift in the configuration. Ensure all necessary labels are defined in Terraform to avoid unexpected updates.
+
+### Removed
+
+- **BREAKING CHANGE**: We have removed the helm values outputs.
```

Comment:
```suggestion
### Removed

- ⚠️ **BREAKING CHANGES**:
  - we have removed the `lifecycle.ignore_changes` block from the `kubernetes_namespace` resource configuration. External modifications on namespace labels will now cause drift in the configuration. Ensure all necessary labels are defined in Terraform to avoid unexpected updates.
  - We have removed the helm values outputs.
```